modules/RTC/JitsiLocalTrack.js

```diff
@@ -524,30 +514,19 @@ export default class JitsiLocalTrack extends JitsiTrack {
                 facingMode: this.getCameraFacingMode()
             };
 
-            if (browser.usesNewGumFlow()) {
-                promise
-                    = RTCUtils.newObtainAudioAndVideoPermissions(Object.assign(
+            if (this.resolution) {
+                streamOptions.resolution = this.resolution;
+            }
+            promise = RTCUtils.newObtainAudioAndVideoPermissions(Object.assign(
```

Comment:
maybe drop the `new` from the name?

Reply:
Sure, I will drop the "new" prefix from all the APIs.